00000-migrate-enarx-public-chat/README.md

```diff
@@ -1,8 +1,9 @@
 # 00000-migrate-enarx-public-chat
 - Authors: Mark Bestavros, Axel Simon
-- Status: [PROPOSED](/README.md#proposed)
+- Status: [DEMONSTRATED](/README.md#demonstrated)
```

Comment:
Status changes must appear in their own PR containing nothing but the status change.